scripts/generator/graphql_generator/csharp/SDK/Android/AndroidWebCheckout.cs.erb

```diff
@@ -0,0 +1,45 @@
+namespace <%= namespace %>.SDK.Android {
+    #if UNITY_ANDROID
+    using System;
+    using <%= namespace %>.SDK;
+
+    #if !SHOPIFY_MONO_UNIT_TEST
+    using UnityEngine;
+    #endif
+
+    class AndroidWebCheckout: WebCheckout {
+
+        private ShopifyClient _client;
+        private Cart _cart;
+
+        protected override ShopifyClient Client {
```

Comment:
Reason being for an abstract class was to reduce the class hierarchy, since we're not making too many of these classes I don't see duplication too much of an issue down the road, but flattening class hierarchy might make it more clear for readers 🤷‍♂️ 

> Would it make more sense to turn IWebCheckout into an abstract base class instead of an interface? At least that way we could flatten the class hierarchy a bit.